testutils/metrics/metrics.go

```diff
@@ -35,9 +35,11 @@ type TestStatsReporter struct {
 	GaugesWG   sync.WaitGroup
 	TimersWG   sync.WaitGroup
 
-	Counters map[string]int64
-	Gauges   map[string]float64
-	Timers   map[string]time.Duration
+	Counters           map[string]int64
+	Gauges             map[string]float64
+	Timers             map[string]time.Duration
+	HistogramSamples   map[string]tally.Buckets
```

Comment:
I meant for Tally 2.0 `CachedStatsReporter`. We don't have its implementation here, but we rely on internal implementation from our private repo.